docs/workforce-management-framework/topics/pro-passportauth-enablement.adoc

```diff
@@ -0,0 +1,64 @@
+[id='{context}pro-passportauth-enablement']
+= Enabling PassportAuth
+
+== Prerequisites
+. Ensure you have completed xref:getting-started[Getting Started]
+
+== Enabling PassportAuth on the {PROJECT_NAME} Applications
```

Comment:
It clashes with enabled by default. Assume that person just started. This statement may be confusing

Reply:
just say if you enqbled keycloak previously

Reply:
I've moved the note for this to the top level of the document